nav2_bringup/launch/bt_navigator.xml

```diff
@@ -0,0 +1,10 @@
+<root main_tree_to_execute="MainTree">
```

Comment:
Is this new BT tree file related to the launch script change or was this not supposed to be in this PR?

Reply:
I added it so that the launch directory had everything needed to launch the stack - the launch script, params file, the map yaml and pgm, and the BT navigator XML file.